content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -50,51 +46,125 @@ section, click :guilabel:`Update info`. From here, fill out the fields:
+   :guilabel:`Exoneration` kind and :guilabel:`Law Reference` filled in.
+
+.. image:: italy/italy-tax-exemption.png
+   :alt: Tax Exemption Settings
 
-Electronic Data Interchange (EDI)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. seealso::
+   There are many :guilabel:`Tax Exemption Kind (Natura)` and :guilabel:`Law Reference` codes.
+   Make sure you check the latest version available to get the latest information on:
 
-Odoo uses the **FatturaPA** :abbr:`EDI (Electronic Data Interchange)` format for the Italian
-localization and is enabled on the default journals when installed. When the **file processing
-authorization** has been set, all **invoices** and **bills** are automatically sent.
+   - `Italian authorities documentation <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
+   - `Official guide on Tax exemption  <https://www.agenziaentrate.gov.it/portale/documents/20143/451259/Guida_compilazione-FE-Esterometro-V_1.9_2024-03-05.pdf/67fe4c2d-1174-e8de-f1ee-cea77b7f5203>`_
 
 .. note::
-   You can :ref:`enable electronic invoicing for other sales and purchase journals
-   <e-invoicing/configuration>` than the default ones.
+   If you need to use a different kind of :guilabel:`Exoneration`, go to :menuselection:`Accounting
+   --> Configuration --> Taxes`, select a similar tax, then click on the cog icon and select
+   :guilabel:`Duplicate`. In the :guilabel:`Advanced Options` tab, add the :guilabel:`Exoneration`
+   and :guilabel:`Law Reference`, to confirm click on :guilabel:`Save`.
+
+.. tip::
+   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
+   differentiate them easily.
 
-You can check the current status of an invoice by the :guilabel:`Electronic invoicing` field. The
-XML file can be found in the **chatter** of the invoice.
+.. _italy/reverse-charge:
 
-.. image:: italy/italy-test.png
-   :align: center
-   :alt: Electronic invoicing status (waiting for confirmation)
+Reverse charge
+==============
 
-.. seealso::
-   :doc:`../accounting/customer_invoices/electronic_invoicing`
+Italian businesses selling goods and services are sometimes required *not* to charge the customer
+for the VAT. The customers pay the VAT *themselves* to the :abbr:`AdE (Agenzia delle Entrate)`
+instead. This mechanism is called **reverse charge**; there are different types:
 
-File processing authorization (Odoo)
-------------------------------------
+- :guilabel:`Internal Reverse Charge` (for domestic sales) is when the VAT is paid by the seller.
+
+- :guilabel:`External Reverse Charge` (for intra-EU sales). VAT is a type of tax that is applied to
+  the country of the buyer. The seller has no means to collect and pay VAT in the other country, so
+  the buyer is responsible for doing it in a B2B setting.
+
+Invoices
+--------
+
+**Reverse-charged** customer invoices are **tax exempt** (0%) for the seller. To use it on an invoice,
```

Comment:
The wording is incorrect. The transaction is not exempt of taxes. The responsibility to declare it is transfered on the buyer.